clang/lib/CIR/CodeGen/CIRGenStmtOpenMP.cpp

```diff
@@ -0,0 +1,45 @@
+//===--- CIRGenStmtOpenMP.cpp - Emit MLIR Code from OpenMP Statements -----===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This contains code to emit OpenMP Stmt nodes as MLIR code.
+//
+//===----------------------------------------------------------------------===//
+
+#include "CIRGenFunction.h"
+#include "mlir/Dialect/OpenMP/OpenMPDialect.h"
+
+using namespace cir;
+using namespace clang;
+using namespace mlir::cir;
+using namespace mlir::omp;
```

Comment:
It's probably best if only pick either `mlir::cir` or `mlir::omp`, but not both at the same time - just in order to prevent ambiguity (for example, both have a `YieldOp`). I'm fine with any of your choice.

Reply:
You haven't address this one.

Reply:
I think the change it's not reflecting because I forced pushed (because I rebased). However, see this file: https://github.com/llvm/clangir/blob/2c4a281348db91da569851e11d2f24ad6985f37c/clang/lib/CIR/CodeGen/CIRGenStmtOpenMP.cpp . I removed `mlir::cir`